sovryn-token-bridge/federator/src/lib/ClientId.js

```diff
@@ -0,0 +1,20 @@
+export const ETHERSCAN_CHAIN_ID = 1
+const web3 = require('web3');
+
+module.exports = class ClientId {
+    constructor(logger, config, Web3 = web3) {
+        this.config = config;
+        this.logger = logger;
+        EtherScanChainId = ETHERSCAN_CHAIN_ID;
```

Comment:
You don't need this variable -- you can just compare to `ETHERSCAN_CHAIN_ID` below in `isEthereumChain()`.

Reply:
Done